templates/common/on-prem/files/NetworkManager-resolv-prepender.yaml

```diff
@@ -48,6 +48,7 @@ contents:
             {{ .Images.baremetalRuntimeCfgImage }} \
             node-ip \
             show \
+            --retry-on-failure \
```

Comment:
Any chance that this causes a delay over the NM timeout?

Reply:
I suppose it's possible, but in the particular case that was triggered by the new runtimecfg behavior it should only take a couple of seconds for the IP to settle and become usable.

I can add a timeout just to be safe, although if that happens we're liable to end up with a missing resolv.conf anyway.